docs/source/release_notes/release-notes.rst

```diff
@@ -42,21 +42,20 @@ v0.12.0
          - Change
 
     *    - |breaking|
-         - If you do not specify a leader block in your config, AtlasDB will now still try to register the timestamp and lock endpoints necessary for other clients or CLIs to run in the same keyspace.
-           This may require changes in setup logic for applications that have previously only ever run with no leader block.
+         - AtlasDB will always try to register timestamp and lock endpoints for your application, whereas previously this only occurs if you specify a :ref:`leader-config`.
+           This ensures that CLIs will always be able to run against your service.
+           Applications that have previously only ever run with no leader block may need to make changes to startup logic.
```

Comment:
I remember some discussion about this, but details escape me - what's the scenario where this is actually breaking? I'd like to include more details about who's affected.

I think @tpetracca went and actually tried this change on various products while trying out the backup stuff - did any of the products fail to startup?